app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -94,36 +78,56 @@ global.ResizeObserver = jest.fn().mockImplementation(() => ({
   disconnect: jest.fn(),
 }));
 
-describe('Week', () => {
-  const teamSelect = 'Browns';
-  const NFLTeams = [{ teamName: 'Browns', teamId: '1234', teamLogo: 'browns' }];
-  const user = { id: '12345', email: 'email@example.com', leagues: [] };
-  const entry = 'mockEntry';
-  const league = 'mockLeague';
-  const week = '1';
-  const setUserPick = jest.fn();
-  const updateWeeklyPicks = jest.fn();
-  const mockGetCurrentLeague = getCurrentLeague as jest.Mock;
-  const mockCreateWeeklyPicks = createWeeklyPicks as jest.Mock;
-
-  const weeklyPicks: IWeeklyPicks = {
-    leagueId: '123',
-    gameWeekId: '123456',
-    userResults: {},
-  };
+// constants
+const teamSelect = 'Browns';
+const NFLTeams = [
+  {
+    teamId: 'browns',
```

Comment:
For future ticket: This might need further discussion, but the naming convention here is honestly very confusing considering the teamId value is the actual team name

Reply:
@choir27 yeah, I can totally see that.  This is how it is actually setup in the Appwrite DB.

![CleanShot 2024-09-17 at 09 47 50](https://github.com/user-attachments/assets/e918e8f0-9cca-4846-8b9b-8a55ebbc1d16)